state/store.go

```diff
@@ -23,18 +24,49 @@ const (
 	valSetCheckpointInterval = 100000
 )
 
+const (
+	prefixValidators       = int64(5)
+	prefixConsensusParams  = int64(6)
+	prefixAllABCIResponses = int64(7)
+	prefixState            = int64(8)
+	prefixLastABCIResponse = int64(13)
+)
```

Comment:
You need to use all the old prefixes and key encodings

Reply:
Else this breaks v0.34.x

Reply:
We should probably even have tests that ensure keys don't change throughout a release